code/CMakeLists.txt

```diff
@@ -89,9 +89,9 @@ if(BuildSPEngine OR BuildJK2SPEngine)
 			${OpenJKLibDir}/SDL2/include
 			)
 	else()
-		find_package(SDL2 REQUIRED)
-		set(SPEngineIncludeDirectories ${SPEngineIncludeDirectories} ${SDL2_INCLUDE_DIR})
-		set(SPEngineLibraries ${SPEngineLibraries} ${SDL2_LIBRARY})
+		find_package(SDL2 REQUIRED CONFIG)
```

Comment:
> This requires SDL >= 2.0.4.

This breaks the Travis-CI build, because Travis-CI is using Ubuntu 14.04, which only has SDL 2.0.2.

I'll add a commit to bump it up to Ubuntu 18.04, which is the latest LTS, unless you particularly care about older versions? (16.04 just about has a new enough SDL.)

Reply:
Branch updated. Let me know if you'd prefer 16.04 (but it only has about a year of support lifetime left so maybe not).